app/views/users/_user.html.slim

```diff
@@ -2,12 +2,12 @@
  .user-index-size
   p
     - if user.avatar?
-      = image_tag(user.avatar.url, alt: 'Image', class: 'user-avatar')
+      = user_avatar_tag(user)
     - else
-      = image_tag("default_a.png", alt: 'Image', class: 'user-avatar')
+      = user_avatar_tag(user)
```

Comment:
you already have an if-else statement inside the helper method
```
p = user_avatar_tag(user)
```

also `<p></p>` is a text tag, not the best choice for an image wrapper

Reply:
done